volunteer-app/www/js/app.js

```diff
@@ -35,10 +35,10 @@
 				debug: false,
 				environment: 'live',	// dev | stage | live
 				apiBaseUrl: {
-					local: 'http://localhost:4000/v1/',
-					dev:   'http://localhost:4000/v1/',
-					stage: 'https://twine-api-staging.herokuapp.com/v1/',
-					live:  'https://api.twine-together.com/v1/',
+					local: 'http://localhost:4000/v1.1/',
```

Comment:
This app should be staying on `v1`. This is the reason for the minor bump.

Reply:
I know, but until the new native app is hooked up to the API, we have not decent way of testing the v1.1 projects API beyond `curl`ing the server. So I figured I'd upgrade here too. Can revert before merging.

Reply:
Can you add comments/an issue as a reminder? I wouldn't want this to get forgotten about and merged with this feature

Reply:
Added to top of PR description

Reply:
Oh my mistake, as I was set as a reviewer I assumed this PR was ready n you were waiting for tests

Reply:
On second view, I may be just be giving an unrequested review. My bad.

Reply:
🤷🏼‍♂️ it was basically ready from a code perspective I just hadn't tested whether anything actually worked yet.